packages/core/package.json

```diff
@@ -137,7 +137,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "resize-observer-polyfill": "^1.5.1",
-    "sass": "^1.64.2",
+    "sass": "~1.64.2",
```

Comment:
Prevent core from moving up to sass@1.65.x due to bug in @carbon/styles (until issue is resolved).